lnbits/templates/base.html

```diff
@@ -315,19 +315,21 @@
         window.allowedThemes = themes.map(str => str.trim())
       }
       window.langs = [
-        { value: 'en', label: 'English', display: 'EN' },
-        { value: 'de', label: 'Deutsch', display: 'DE' },
-        { value: 'es', label: 'Español', display: 'ES' },
-        { value: 'jp', label: '日本語', display: 'JP' },
-        { value: 'cn', label: '中文', display: 'CN' },
-        { value: 'fr', label: 'Français', display: 'FR' },
-        { value: 'it', label: 'Italiano', display: 'IT' },
-        { value: 'pi', label: 'Pirate', display: 'PI' },
-        { value: 'nl', label: 'Nederlands', display: 'NL' },
-        { value: 'we', label: 'Cymraeg', display: 'CY' },
-        { value: 'pl', label: 'Polski', display: 'PL' },
-        { value: 'pt', label: 'Português', display: 'PT' },
-        { value: 'br', label: 'Português', display: 'BR' }
+        { value: 'en', label: 'English', display: '🇬🇧 EN' },
+        { value: 'de', label: 'Deutsch', display: '🇩🇪 DE' },
+        { value: 'es', label: 'Español', display: '🇪🇸 ES' },
+        { value: 'jp', label: '日本語', display: '🇯🇵 JP' },
+        { value: 'cn', label: '中文', display: '🇨🇳 CN' },
+        { value: 'fr', label: 'Français', display: '🇫🇷 FR' },
+        { value: 'it', label: 'Italiano', display: '🇮🇹 IT' },
+        { value: 'pi', label: 'Pirate', display: '🏴‍☠️ PI' },
+        { value: 'nl', label: 'Nederlands', display: '🇳🇱 NL' },
+        { value: 'we', label: 'Cymraeg', display: '🏴󠁧󠁢󠁷󠁬󠁳󠁿 CY' },
+        { value: 'pl', label: 'Polski', display: '🇵🇱 PL' },
+        { value: 'pt', label: 'Português', display: '🇵🇹 PT' },
+        { value: 'br', label: 'Português do Brasil', display: '🇧🇷 BR' },
+        { value: 'cs', label: 'Česky', display: '🇨🇿 CS' },
+        { value: 'sk', label: 'Slovensky', display: '🇸🇰 SK' }
```

Comment:
Should the languages be ordered by the international (two letter) language code (they look random now)?
... `EN`, `DE`, `ES` ... -> ... `DE`, `EN`, `ES` ...

Maybe very popular languages like `EN` & `ES` can stay at the top.

Reply:
I prefer to do that in another PR so we avoid the bikeshedding here and can have these merged soon.